core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -175,20 +176,20 @@ def run_migrations
       end
     end
 
+    def install_subcomponents
+      apply_template_for :authentication, @selected_authentication
+      apply_template_for :frontend, @selected_frontend
+      apply_template_for :payment_method, @selected_payment_method
+    end
+
     def install_solidus_admin
       return unless options[:admin_preview]
 
       say_status :installing, "SolidusAdmin", :blue
       unless File.read(app_path.join('Gemfile')).include?('solidus_admin')
         bundle_command 'add solidus_admin -v ">= 0.2"'
       end
-      generate 'solidus_admin:install'
-    end
-
-    def install_subcomponents
-      apply_template_for :authentication, @selected_authentication
-      apply_template_for :frontend, @selected_frontend
-      apply_template_for :payment_method, @selected_payment_method
+      generate "solidus_admin:install #{'--tailwind' if options[:build_admin_tailwind]}"
```

Comment:
Sorry for the back and forth (and it might not be related to this PR), but at this point I don't understand in which case we don't need to install Tailwind along with the new admin. I mean, in which circumstances `build_admin_tailwind` will be set to false?

Reply:
Right now, never. But if for any reason somebody wants to disable it, we give the possibility to do it with `rails g solidus:install --no-build-admin-tailwind`, for example.